app/views/main/php_search.haml

```diff
@@ -1,35 +0,0 @@
-= javascript_include_tag :partial_replacer
-
-:javascript
-  function addText(id, text) {
-    document.getElementById("box_" + id.toString()).value = text;
-  }
-
-- if @result_count == 0
-  %p
-    No search results for #{content_tag :span, h(params[:query]), style: 'color: #f00; font-weight: bold;'} :(
-- else
-  - if @requester_count > 1
-    %p#advisory.red.bigger
-      Your search returned reviews of #{@requester_count} different requesters. If you are leaving a review, please be sure to review the right one! Thank you!
```

Comment:
do we still want this message?

Reply:
![image](https://user-images.githubusercontent.com/10545461/126884204-b73afb56-7b97-468e-8929-7900a5ab1ab9.png)

The query to get the count unfortunately takes about 400 ms but it's not the slowest thing on the page by far